From 7ece2b8a8fed13d5e645097bc3432aef84a04ef0 Mon Sep 17 00:00:00 2001
From: Andy CrossGate Yan <GeForce8800Ultra@gmail.com>
Date: Tue, 11 Aug 2026 09:46:45 +0000
Subject: [PATCH] SystemUI: monet: Remove unused colorSchemeIsApplied

[Port to A16] Method body drifted upstream (MaterialDynamicColors ctor, mDarkColorScheme); whole-method removal applied to A16 version.

Change-Id: Id9a0b0426d423912972a4c61dea8a0499d821136
---
 .../theme/ThemeOverlayController.java         | 42 -------------------
 1 file changed, 42 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java b/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java
index c3ce071e6ed3..553f18512fcc 100644
--- a/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java
+++ b/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java
@@ -747,48 +747,6 @@ public class ThemeOverlayController implements CoreStartable, Dumpable {
         });
     }
 
-    /**
-     * Checks if the color scheme in mColorScheme matches the current system palettes.
-     *
-     * @param managedProfiles List of managed profiles for this user.
-     */
-    private boolean colorSchemeIsApplied(Set<UserHandle> managedProfiles) {
-        final ArraySet<UserHandle> allProfiles = new ArraySet<>(managedProfiles);
-        allProfiles.add(UserHandle.SYSTEM);
-        for (UserHandle userHandle : allProfiles) {
-            Resources res = userHandle.isSystem()
-                    ? mResources : mContext.createContextAsUser(userHandle, 0).getResources();
-            Resources.Theme theme = mContext.getTheme();
-            MaterialDynamicColors dynamicColors = new MaterialDynamicColors();
-            if (!(res.getColor(android.R.color.system_accent1_500, theme)
-                    == mColorScheme.getAccent1().getS500()
-                    && res.getColor(android.R.color.system_accent2_500, theme)
-                    == mColorScheme.getAccent2().getS500()
-                    && res.getColor(android.R.color.system_accent3_500, theme)
-                    == mColorScheme.getAccent3().getS500()
-                    && res.getColor(android.R.color.system_neutral1_500, theme)
-                    == mColorScheme.getNeutral1().getS500()
-                    && res.getColor(android.R.color.system_neutral2_500, theme)
-                    == mColorScheme.getNeutral2().getS500()
-                    && res.getColor(android.R.color.system_outline_variant_dark, theme)
-                    == dynamicColors.outlineVariant().getArgb(mDarkColorScheme.getMaterialScheme())
-                    && res.getColor(android.R.color.system_outline_variant_light, theme)
-                    == dynamicColors.outlineVariant().getArgb(mLightColorScheme.getMaterialScheme())
-                    && res.getColor(android.R.color.system_primary_container_dark, theme)
-                    == dynamicColors.primaryContainer().getArgb(
-                    mDarkColorScheme.getMaterialScheme())
-                    && res.getColor(android.R.color.system_primary_container_light, theme)
-                    == dynamicColors.primaryContainer().getArgb(
-                    mLightColorScheme.getMaterialScheme())
-                    && res.getColor(android.R.color.system_primary_fixed, theme)
-                    == dynamicColors.primaryFixed().getArgb(
-                    mLightColorScheme.getMaterialScheme()))) {
-                return false;
-            }
-        }
-        return true;
-    }
-
     @SuppressWarnings("StringCaseLocaleUsage") // Package name is not localized
     private void updateThemeOverlays() {
         final int currentUser = mUserTracker.getUserId();
-- 
2.43.0

